packages/plugin-atoms/src/components/Loader.tsx

```diff
@@ -0,0 +1,66 @@
+import { useComponentUtils } from '@ima/react-page-renderer';
+import {
+  useEffect,
+  useRef,
+  useState,
+  memo,
+  ComponentPropsWithoutRef,
+} from 'react';
+
+export const Loader = memo(function LoaderComponent({
+  timeout,
+  center,
+  color,
+  className,
+  ...rest
+}: {
+  timeout?: number;
+  center?: boolean;
+  color?: string;
```

Comment:
nemělo by tu být `black` | `white` když to žádné jiné barvy neumí? Rozumím možnosti rozšířit u sebe v projektu, ale teď to úplně typově nesedí

Reply:
fixed